src/components/continue/continue.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component, Fragment } from 'react';
```

Comment:
The folder name should start with an Uppercase letter and all files with `jsx` should have the `jsx` extension. This is based on The Team Coding Conventions we signed up for